configurable_lti_consumer/configurable_lti_consumer.py

```diff
@@ -1,64 +1,123 @@
+import re
+
 from django.conf import settings
 from lti_consumer import LtiConsumerXBlock
 
-from .defaults import DEFAULT_CONFIGURABLE_LTI_CONSUMER_SETTINGS
 from .exceptions import ConfigurableLTIConsumerException
 
 
 class ConfigurableLtiConsumerXBlock(LtiConsumerXBlock):
-    def __init__(self, *args, **kwargs):
-        super(ConfigurableLtiConsumerXBlock, self).__init__(*args, **kwargs)
+    """Overriding the LTI consumer XBlock to make it configurable from Django settings."""
+
+    @property
+    def editable_fields(self):
+        """
+        Remove, from editable fields, the fields that we have already configured in settings.
 
-        if not self._runtime.__class__.__name__ == "ImportSystem":
-            # We do not need this when importing course in studio
-            self.set_editable_field_names()
+        The `launch_url` field should always be in editable fields so our setting configurations
+        are only overrides but not "typing" for an LTI consumer Xblock. To say it differently,
```

Comment:
```suggestion
        are only overridden but not "typing" for an LTI consumer Xblock. To say it differently,
```
I don't get what you mean by "typing", do you mean "typed"?

Reply:
I changed the whole sentence. Can you review the last fixup commit?